web/src/click_handlers.ts

```diff
@@ -838,7 +838,7 @@ export function initialize(): void {
 
     // LEFT SIDEBAR
 
-    $("body").on("click", "#clear_search_topic_button", topic_list.clear_topic_search);
+    $("body").on("click", ".filter-topics .input-button", topic_list.clear_topic_search);
```

Comment:
This is not a good selector -- previously, it was very clear that this was the "clear topic" button, now it's just "whatever input-button is inside this input".

We can fix this in a follow-up, but we should add a class to the close button that selectors like this can target.